src/Dropdown.js

```diff
@@ -20,9 +20,6 @@ function Dropdown({
   onChange,
   onSelect,
   baseClassName,
-  controlClassName,
-  placeholderClassName,
-  menuClassName,
   arrowClassName,
```

Comment:
why keep the arrowClassName ?

why keep the baseClassname as a prop when the readme says its no longer configurable? maybe should jus be referenced as a constant